src/components/loader.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import pageLoader from 'images/load.gif';
+import searchLoader from 'images/Spinner.gif';
+
+export function PageLoader ( {state} ) {
+  return state ?(
+    <div className="loader-bg" >
+      <img className="loader" src={pageLoader} alt="" />
+    </div>
+  ) : <div></div>
+}
+
+export function SearchLoader({ loading }) {
```

Comment:
Same goes for SearchLoader, you will render it or not in a component where it's called depending on the inner state of that component. No need for SearchLoader to receive any props.